docs/next.config.js

```diff
@@ -24,6 +24,14 @@ console.log(`Using React '${reactMode}' mode.`);
 const l10nPRInNetlify = /^l10n_/.test(process.env.HEAD) && process.env.NETLIFY === 'true';
 const vercelDeploy = Boolean(process.env.VERCEL);
 
+const staging =
+  process.env.REPOSITORY_URL === undefined ||
+  /mui-org\/material-ui$/.test(process.env.REPOSITORY_URL);
```

Comment:
This seems to exclude `mui-org/material-ui-x` ($)

Reply:
That's intended. I cannot know how this code behaves in a separate repo. You'd have to enable this behavior in your repository not here.

Reply:
My bad, I didn't notice we were inside next.config.js